cf-oscore/src/main/java/org/eclipse/californium/cose/EncryptCommon.java

```diff
@@ -97,7 +104,12 @@ void encryptWithKey(byte[] rgbKey) throws CoseException, IllegalStateException {
 	private int getAES_CCM_IVSize(AlgorithmID alg) throws CoseException {
```

Comment:
Now the difference to the new `public static int ivLength(AlgorithmID alg)` seems to be very small. Maybe you redesign it a little to remove this one?

Reply:
True, they are very similar now. I will redesign things to only keep 1.

Just to clarify, you mean removing getAES_CCM_IVSize and keeping ivLength, or the opposite?

Reply:
I guess, the `public static` one is the one to keep. if you keep the -1 or the exception depends on the calling contexts. Just check, what will fit better for you.